### ISSUE TITLE
Setting options after widget creation

### ISSUE THREAD

**Issue by sdrobs** [yaronn/blessed-contrib#99]
I see that the lcd display has a setOptions function to adjust options at runtime. It would be nice to have this functionality for other widgets as well. One use case I have in min is dynamically adjusting the minY and maxY for line charts in order to achieve a zooming effect as described in #86 .

Interested in hearing your thoughts on this, thanks!

**Comment by thomasdondorf** [comment]
I agree that a `setOptions` for all widgets would be nice. But you can also do it without it. Just remove the widget from the screen and recreate it.

**Example**
You create the line chart like this:
```
line = contrib.line(options);
screen.append(line);
```

Then, if you want to change an option like `maxY`:
```
screen.remove(line);
options.maxY = 10000;
line = contrib.line(options);
screen.append(line);
screen.render();
```

This works as well if you are using a grid (as the grid will just add the object to the screen). I assume it works the same for the lcd display.
